### ISSUE TITLE
Implement ArgoCD synchronization failure notifications

### ISSUE THREAD

**Issue by jleach** [bcgov/platform-services-registry#221]
## Story

As a P.S. team member I would like to be notified when ArgoCD synchronizations fail.

## Description

When an ArgoCD sync fails, I would like to get a notification in a RocketChat channel.

## Done When

Status message are posted to the appropriate RocketChat channel(s) following a sync failure in ArgoCD.

**Comment by j-pye** [comment]
We're currently blocked on this due to an ArgoCD version limitation. I'm actively watching the ArgoCD Notification application repo for an update that can support older versions of ArgoCD. If we upgrade to a newer version of ArgoCD this will no longer be a blocker.

**Comment by IanKWatts** [comment]
The KLAB cluster has been configured to support ArgoCD notifications.

A document has been added to the platform-services-docs repo with instructions on configuring a cluster and enabling notifications for resources.
https://github.com/bcgov-c/platform-services-docs/blob/main/argocd-notifications.md 

There isn't a request for notifications to be enabled for any particular resource, so I think this satisfies the requirements of this task.  I will create a separate task to have the other clusters configured.  There is already a separate task for the setup of notifications for Argo Workflows.  We're holding off on that one for the time being, because Workflows will be upgraded soon.
